core-cart.mjs

```diff
@@ -2,9 +2,90 @@ import {carts, products, promoCode, users} from "./dataset.mjs";
 
 import * as fs from "fs";
 import * as os from "os";
+import { moveMessagePortToContext } from "worker_threads";
 
-const discountedPrice = (price, rate = 0.10) => (price * (1 - rate)).toFixed(2);
+//INTESTAZIONE SCONTRINO
+const heading = function(){
+    let nMacchina='NOMEMACCHINA Cart - 43874\n'
+    let dateFormat = new Intl.DateTimeFormat('en-US',{
```

Comment:
se metti timezone Europe/Rome usa `it-IT` come formato!
Lo stesso output che ottieni in 7 righe  è ottenibile in una `new Date().toDateString();`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toDateString